src/plugins/timeConductor/ConductorHistory.vue

```diff
@@ -0,0 +1,161 @@
+            <span class="c-button__label">History</span>
+        </button>
+        <div class="c-menu" v-if="open">
+            <ul
+                v-if="isUTCBased"
+            >
+                <li @click="selectHours(24)">Last 24 hours</li>
+                <li @click="selectHours(2)">Last 2 hours</li>
+                <li @click="selectHours(1)">Last hour</li>
+            </ul>
+            <ul>
+                <li
+                    v-for="(tick, index) in historyForCurrentTimeSystem"
+                    :key="index"
+                    @click="selectTick(tick)"
+                >
+                    {{ formatTime(tick.start) }} - {{ formatTime(tick.end) }}
+                </li>
+            </ul>
+        </div>    
+    </div>
+</template>
+
+<style lang="scss">
+    @import "~styles/sass-base";
+</style>
+
+<script>
+import toggleMixin from '../../ui/mixins/toggle-mixin';
+import utcMultiTimeFormat from './utcMultiTimeFormat.js';
+import _ from 'lodash'
+
+const LOCAL_STORAGE_HISTORY_MAX_RECORDS = 20;
+const LOCAL_STORAGE_HISTORY_KEY = 'tcHistory';
+
+export default {
+    inject: ['openmct'],
+    mixins: [toggleMixin],
+    props: {
+        bounds: {
+            type: Object,
+            required: true
+        },
+        timeSystem: {
+            type: Object,
+            required: true
+        }
+    },
+    data() {
+        return {
+            history: {} // contains arrays of ticks {start, end}, array key is time system key
+        }
+    },
+    computed: {
+        isUTCBased() {
+            return this.timeSystem.isUTCBased;
+        },
+        historyForCurrentTimeSystem() {
+            return this.history[this.timeSystem.key]
+        }
+    },
+    methods: {
+        getHistoryFromLocalStorage() {
+            if (localStorage.getItem(LOCAL_STORAGE_HISTORY_KEY)) {
+                this.history = JSON.parse(localStorage.getItem(LOCAL_STORAGE_HISTORY_KEY))
+            } else {
+                this.history = {};
+                this.persistHistoryToLocalStorage();
+            }
+        },
+        persistHistoryToLocalStorage() {
+            localStorage.setItem(LOCAL_STORAGE_HISTORY_KEY, JSON.stringify(this.history));
+        },
+        addTick() {
+            const key = this.timeSystem.key;
+            let [...currentHistory] = this.history[key] || [];
+            const tick = {
+                start: this.bounds.start,
+                end: this.bounds.end,
+            };
+
+            // when choosing an existing entry, remove it and add it back as latest entry
+            currentHistory = currentHistory.filter((entry) => {
+                return !_.isEqual(tick, entry);
+            });
+
+            if (currentHistory.length >= LOCAL_STORAGE_HISTORY_MAX_RECORDS) {
+                currentHistory.shift();
+            }
+
+            currentHistory.push(tick);
+            this.history[key] = currentHistory;
+        },
+        selectTick(tick) {
```

Comment:
This nomenclature might cause some confusion. Typically when we refer to time axis "ticks" we're referring to the vertical lines on the axis. It looks as though what we're referring to here are entires in the bounds history?